networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/ActiveEth2P2PNetworkTest.java

```diff
@@ -255,6 +259,44 @@ void onSyncStateChanged_shouldNotResultInMultipleSubscriptions() {
     verify(eventChannels, times(1)).subscribe(eq(BlockGossipChannel.class), any());
   }
 
+  @Test
+  void isCloseToInSync_shouldCalculateWhenDistanceOutOfRange() {
+    // Current slot is a long way beyond the chain head
```

Comment:
because the fn name is good, we can nuke the comments on both, now that variable names etc are descriptive.